### PR TITLE
Add filter and label example to fluent.conf. fix #512

### LINKED ISSUE #512: update the default config file to include filters and labels
The current example config generated by `--setup` does not include a filter or label example.

- I feel pretty strongly about having a couple of filter examples.
- I am less concerned about labels since it's more "advanced" and not needed until more complex event routing is required. but it probably doesn't hurt to have an example toward the bottom of `fluent.conf`.

### PR DESCRIPTION
**Pull request by repeatedly**
@kiyoto could you check?

### CODE REVIEW

**Comment by repeatedly** [comment]
Or @sonots @tagomoris how  about this?

File: fluent.conf [hunk 1]
```diff
@@ -31,6 +31,24 @@
...
+#    hostname ${hostname}
+#    tag ${tag}
+#  </record>
+#</filter>
+
+## Selecting event filter
+## Remove unnecessary events from apache prefixed tag events
+#<filter apache.**>
+#  @grep
+#  include1 method GET # pass only GET in 'method' field
```
Comment (sonots) [inline fluent.conf]: regexp1

**Comment by sonots** [comment]
Need an exmple to emit messages with label (source)

**Comment by repeatedly** [comment]
@sonots Please commit changes to this branch directly :)

File: fluent.conf [hunk 2]
```diff
@@ -31,6 +31,24 @@
 #  tag apache.access
 #</source>
 
+## Mutating event filter
+## Add hostname and tag fields to apache.access tag events
+#<filter apache.access>
+#  @record_transformer
```
Comment (tagomoris) [inline fluent.conf]: `@type record_transformer`?

File: fluent.conf [hunk 3]
```diff
@@ -31,6 +31,24 @@
...
+#  <record>
+#    hostname ${hostname}
+#    tag ${tag}
+#  </record>
+#</filter>
+
+## Selecting event filter
+## Remove unnecessary events from apache prefixed tag events
+#<filter apache.**>
+#  @grep
```
Comment (tagomoris) [inline fluent.conf]: ditto

**Comment by repeatedly** [comment]
Fix typo of missing `@type`

**Comment by repeatedly** [comment]
Merged

**Pull request merged**